From 349a4fcb3c303928a433ceabb2b09b47c5a56a35 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sun, 13 Mar 2022 18:14:34 +0800
Subject: [PATCH 2/3] driver: pvr_drm: don't set dev_priv_size = 0

The new version kernel has moved the case "dev_priv_size",
and the compiler already clears this for us.

Upstream-Status: Inappropriate [upstream is still using k5.10 version]

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 kernel/drivers/staging/imgtec/pvr_drm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/drivers/staging/imgtec/pvr_drm.c b/kernel/drivers/staging/imgtec/pvr_drm.c
index e1ceb91..c51ed4f 100644
--- a/kernel/drivers/staging/imgtec/pvr_drm.c
+++ b/kernel/drivers/staging/imgtec/pvr_drm.c
@@ -281,7 +281,6 @@ static const struct file_operations pvr_drm_fops = {
 const struct drm_driver pvr_drm_generic_driver = {
 	.driver_features	= DRIVER_MODESET | DRIVER_RENDER,
 
-	.dev_priv_size		= 0,
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0))
 	.load			= NULL,
 	.unload			= NULL,
-- 
2.35.1

